hostprocess/flannel/kube-proxy/start.ps1

```diff
@@ -61,7 +61,7 @@ Write-Host "sourceip: $vip"
 
 $arguements = "--v=6",
         "--hostname-override=$env:NODE_NAME",
-        "--feature-gates=WinOverlay=true,IPv6DualStack=false",
+        "--feature-gates=WinOverlay=true",
```

Comment:
i think this fixes this old issue:
https://github.com/kubernetes-sigs/sig-windows-tools/pull/147

also the FG is now on by default (GA).